contracts/fast/FastTokenFacet.sol

```diff
@@ -622,25 +583,19 @@ contract FastTokenFacet is AFastFacet, IERC20 {
    *    - If the fast is semi-public,
    *      - We require that candidate is a member of the Marketplace contract.
```

Comment:
... and that they are active.

Reply:
This is immensely useful for you to have my back :D